### PR TITLE
Dhfprod 1095

### PR DESCRIPTION
**Pull request by aebadirad**
This satisfies the bug, however, xml traces that use json sources still show up blank when being returned. They save properly, they just are mistranslated in the json transformation.

### CODE REVIEW

File: marklogic-data-hub/src/main/resources/scaffolding/harmonize/sjs/main.sjs [hunk 1]
```diff
@@ -1,6 +1,6 @@
 // dhf.sjs exposes helper functions to make your life easier
 // See documentation at:
-// https://github.com/marklogic/marklogic-data-hub/wiki/dhf-lib
+// https://marklogic.github.io/marklogic-data-hub/docs/server-side/
```
Comment (grechaw) [inline marklogic-data-hub/src/main/resources/scaffolding/harmonize/sjs/main.sjs]: oof good catch

File: marklogic-data-hub/src/server-side/impl/trace-lib.xqy [hunk 2]
```diff
@@ -302,7 +302,7 @@ declare %private function trace:write-error-trace(
             "/" || $trace/*:trace/*:traceId || $extension,
             $trace,
             xdmp:default-permissions(),
-            ("trace", $trace/*:trace/*:type)
+            ("trace", $trace/*:trace/*:flow-type, $trace/*:trace/*:flowType)
```
Comment (grechaw) [inline marklogic-data-hub/src/server-side/impl/trace-lib.xqy]: This looks like something we missed during a name change or format introduction.  time to introduce some ml-unit-test?

Comment (grechaw) [inline marklogic-data-hub/src/server-side/impl/trace-lib.xqy]: Two test failures -- assertions around binary nodes.  this bugfix looks legit, the tests' output has changed in what looks like a non-problematic way.  but they need to change to do the merge

**Comment by ayuwono** [comment]
Application is not hang anymore with invalid credentials

But can we change the error message?
Response with status: 401 OK for URL: http://localhost:8080/api/login

**Comment by ayuwono** [comment]
It's good now with the latest commit

**Pull request merged**